src/main/resources/com/google/api/codegen/nodejs/version_index.snip

```diff
@@ -49,10 +49,28 @@
     {@comments(@service.doc.lines)}
     *
     * @@param {object=} options - [Configuration object](#/docs).
+    * @@param {string=} options.key - API key to use form communication with the
+    *     service.
+    * @@param {string=} options.keyFilename - Path to a key file to use for an
+    *     API key.
+    * @@param {(string|number)=} options.logLevel - An integer from 0-5
```

Comment:
I did a simple search through the toolkit and gax-nodejs for "logLevel" and found nothing. Is this option api-specific?